azext_iot/digitaltwins/params.py

```diff
@@ -433,3 +433,91 @@ def load_digitaltwins_arguments(self, _):
     with self.argument_context("dt network private-endpoint connection wait") as context:
         context.ignore("created")
         context.ignore("exists")
+
+    with self.argument_context("dt data-history") as context:
+        context.argument(
+            "conn_name",
+            options_list=["--conn-name", "--cn"],
+            help="Name of time series database connection.",
+            arg_group="TimeSeries Connection",
+        )
+
+    with self.argument_context("dt data-history create adx") as context:
+        context.argument(
+            "adx_cluster_name",
+            options_list=["--adx-cluster-name", "--adxc"],
+            help="Name of Azure Data Explorer cluster to integrate with.",
+            arg_group="Azure Database Explorer Arugments",
+        )
+
+        context.argument(
+            "adx_database_name",
+            options_list=["--adx-database-name", "--adxd"],
+            help="Name of Azure Data Explorer database to integrate with.",
+            arg_group="Azure Database Explorer Arugments",
```

Comment:
Arguments spelling issue, but more importantly that word should be removed as its already added by CLI core.

![image](https://user-images.githubusercontent.com/28658112/151639350-cdc3dcc7-a805-43d6-a628-5552f069c60a.png)

Reply:
`arg_group="Azure Data Explorer",`